### DISCUSSION TITLE
新版本的pikpak多了些东西，

### DISCUSSION THREAD

**Discussion by jinmeng3** [AlistGo/alist#7192] [General]
需要刷新令牌，验证码，ID,这些东西咋获得啊

**Comment by greatgithubfish** [comment]
> 需要刷新令牌，验证码，ID,这些东西咋获得啊

同问，主要是令牌怎么获取，其他都是默认

**Reply by jinmeng3** [reply]
估计是要到network中看，但不知道那个是令牌？

**Comment by zhuliang21** [comment]
> 需要刷新令牌，验证码，ID,这些东西咋获得啊

https://github.com/alist-org/alist/pull/7024#issue-2470412018 这里提供了令牌获取方法
